webui/react/src/shared/utils/routes.test.ts

```diff
@@ -119,10 +119,21 @@ describe('Routes Utilities', () => {
   });
 
   describe('openBlank', () => {
+    let originalWindowOpen: (
+      url?: string | URL,
+      target?: string,
+      features?: string,
+    ) => WindowProxy | null;
```

Comment:
there might be a better type here so open to ideas here

Reply:
`typeof window['open']`?